### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by ciremoussadia**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,34 +8,60 @@ on:
...
     runs-on: ubuntu-latest
-
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x, 15.x]
-
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
     steps:
+      # - uses: act10ns/slack@v1
```
Comment (llambeau) [inline .github/workflows/tests.yml:18]: don't forget to uncomment before merging

**Review by llambeau — CHANGES_REQUESTED** [review]
Bring also the workflows for release etc, like for other projects

**Review by blambeau — COMMENTED** [review]
@llambeau looks good to me. Could you have a quick check too please?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      - uses: act10ns/slack@v1
+        with:
+          status: starting
+          channel: '#opensource-cicd'
+
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
```
Comment (llambeau) [inline .github/workflows/release.yml:67]: Why are we using different node versions for Npm vs Github?

**Review by llambeau — APPROVED** [review]

File: .github/workflows/release.yml (hunk 2 above)
Comment (ciremoussadia) [reply .github/workflows/release.yml:67]: @llambeau I missed it.

**Pull request merged**